lang/en/website-notice.yaml

```diff
@@ -0,0 +1,12 @@
+banner:
+  title: Bitcoincash.org and its owners no longer accurately represent Bitcoin Cash!
+  sub: Click to find out more
+
+title: Notice to services that use Bitcoin Cash (BCH)
+h1: <span class="text-muted">Bitcoincash.org</span> is unreliable for information regarding the BCH network
+h2: Recommended actions
+p1: Due to the decentralized nature of the BCH network, there is no official reference website for general Bitcoin Cash information. Unfortunately, the most popular of those informational websites, bitcoincash.org, has been publishing information that does not reflect the current reality of the BCH network and the November 15th 2020 scheduled upgrade.
+p2: Bitcoin ABC, which owns bitcoincash.org, wants to implement an 8% miner tax. This proposal has been rejected by the vast majority of the Bitcoin Cash ecosystem, and Bitcoin ABC will likely fork to a minority chain as a new coin. This reality has already been recognized in statements from Bitgo, Coinbase, Kraken, and others. Unfortunately, Bitcoin ABC is still presenting its software as the only compatible option for the November 15th 2020 upgrade.
```

Comment:
This invites correction instead of deligitimization. It should be removed or adjusted. I didn't think of anything immediately to replace it so I recommend removing it.

> Unfortunately, Bitcoin ABC is still presenting its software as the only compatible option for the November 15th 2020 upgrade.